components/demo/code-view-styles.js

```diff
@@ -33,7 +34,6 @@ export const styles = css`
 		display: none;
 	}
 	pre[class*="language-"] {
-		border-radius: 5px;
```

Comment:
Having the parent `overflow: hidden` "clips" children when there's a border-radius, which is what this child border-radius was trying to emulate.